src/Dispatcher.cpp

```diff
@@ -264,10 +269,10 @@ void DispatcherBase::DispatchThreadMain() {
         // post outgoing messages if connection is active
         // only send keep-alives on client
         if (conn->state() == NetworkConnection::kActive)
-          conn->PostOutgoing(!m_server);
+          conn->PostOutgoing(m_networkMode & NT_NET_MODE_CLIENT);
```

Comment:
Shouldn't `!m_server` become `!(m_networkMode & NT_NET_MODE_SERVER)`? Otherwise we might have some subtle breaking changes.